### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by EduardoChuks**
Settings added correctly

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/connect/ui/settings/SettingsActivity.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.belatrixsf.connect.ui.settings;
```
Comment (icerrate) [inline app/src/main/java/com/belatrixsf/connect/ui/settings/SettingsActivity.java:21]: License

File: app/src/main/java/com/belatrixsf/connect/ui/settings/SettingsFragment.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package com.belatrixsf.connect.ui.settings;
```
Comment (icerrate) [inline app/src/main/java/com/belatrixsf/connect/ui/settings/SettingsFragment.java:21]: License

File: app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java [hunk 3]
```diff
@@ -28,12 +28,14 @@
...
 import com.belatrixsf.connect.ui.event.EventListActivity;
 import com.belatrixsf.connect.ui.ranking.RankingFragmentListener;
+import com.belatrixsf.connect.ui.settings.SettingsActivity;
 import com.belatrixsf.connect.ui.stars.GiveStarActivity;
 import com.belatrixsf.connect.ui.stars.GiveStarFragment;
 import com.belatrixsf.connect.utils.BelatrixConnectApplication;
 import com.belatrixsf.connect.utils.DialogUtils;
 import com.belatrixsf.connect.utils.di.components.DaggerUserHomeComponent;
 import com.belatrixsf.connect.utils.di.modules.presenters.UserHomePresenterModule;
+import com.twitter.sdk.android.core.models.User;
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java:39]: ?

File: app/src/main/java/com/belatrixsf/connect/ui/home/GuestActivity.java [hunk 4]
```diff
@@ -10,6 +10,7 @@
 import com.belatrixsf.connect.R;
 import com.belatrixsf.connect.ui.about.AboutActivity;
 import com.belatrixsf.connect.ui.event.EventListFragment;
+import com.belatrixsf.connect.ui.settings.SettingsActivity;
 import com.belatrixsf.connect.utils.BelatrixConnectApplication;
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/home/GuestActivity.java:14]: ?

File: app/src/main/res/layout/activity_settings.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
+
+    </android.support.design.widget.AppBarLayout>
+
+    <FrameLayout
+        android:id="@+id/main_content"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
+
```
Comment (PedroCarrillo) [inline app/src/main/res/layout/activity_settings.xml:31]: @icerrate, i think this is a common layout right?

**Pull request merged**